docassemble/RFApackage/data/questions/RFApackage.yml

```diff
@@ -910,27 +971,27 @@ fields:
     
   - note: |
       <h2 class="h3">Access to weapons
-
   - Does the defendant own or have access to guns/firearms or other deadly weapons?: weapons.there_are_any
     input type: radio
     choices:
       - Yes: True
       - No: False
-      - I don't know: None
-
-
-
-
-#  - "Check this box if you have more weapons to list. You can enter them here and it will be attached  as an extra sheet.": extra_sheet_weapons
-#    datatype: yesno
+      - I don't know: None 
+     
+  - "Check this box if you have more weapons to list. You can enter them here and it will be attached as an extra sheet": extra_sheet_weapons
+    datatype: yesno
 #    show if: weapon8
-#    #  if extra_sheet_weapons: 
-#    #    extra_sheets_attached = 1 
-#  - Other weapons: weapons_overflow
-#    input type: area
-#    rows: 4
-#    show if: extra_sheet_weapons
-#continue button field: weapons_info
+  - Other weapons: weapons_overflow
+    input type: area
+    rows: 4
+    show if: extra_sheet_weapons
+continue button field: weapons_info
+---
+code: |
```

Comment:
This is not currently working. It should add an addendum to the affidavit listing the other weapons. I dont think these on screen questions are necessary and should be removed again. Instead, in a code block write if there are more than 8 (?) weapons, trigger the overflow addendum. There is AL documentation on this.